app/actions/ClientActions.js

```diff
@@ -177,6 +178,48 @@ export const getWalletServiceAttempt = () => (dispatch, getState) => {
     .catch(error => dispatch({ error, type: GETWALLETSERVICE_FAILED }));
 };
 
+export const GETTRANSACTIONSSINCELASTOPPENED_ATTEMPT = "GETTRANSACTIONSSINCELASTOPPENED_ATTEMPT";
+export const GETTRANSACTIONSSINCELASTOPPENED_FAILED = "GETTRANSACTIONSSINCELASTOPPENED_FAILED";
+export const GETTRANSACTIONSSINCELASTOPPENED_SUCCESS = "GETTRANSACTIONSSINCELASTOPPENED_SUCCESS";
+
+export const getTransactionsSinceLastOppened = () => async (dispatch, getState) => {
+  dispatch({ type: GETTRANSACTIONSSINCELASTOPPENED_ATTEMPT });
+  const transactionsSinceLastOpenedInfo = {
+    transactionsReceived: [],
+    ticketsVoted:         [],
+    ticketsRevoked:       [],
+    totalReward:           0,
+    totalDCR:              0,
+  };
+  const config = getGlobalCfg();
+  const lastBlockHeightSeen = config.get("last_height");
+  const { currentBlockHeight, walletService, recentTxSinceLastOpenedCount } = getState().grpc;
+
+  try {
+    const { mined, unmined } = await walletGetTransactions(walletService, lastBlockHeightSeen, currentBlockHeight);
```

Comment:
You need to pass `recentTxSinceLastOpenedCount` here.

Reply:
Otherwise, you're still getting the same amount of txs from the wallet, only you're limiting on the client side.

Reply:
Sorry about that. Now I am doing it properly.